scripts/test_pypi_packages.py

```diff
@@ -0,0 +1,87 @@
+"""Usage: python -m scripts.test_pypi_packages"""
```

Comment:
That's a bit tricky. Maybe instead add a `#!` line, make the script executable, and add `sys.path.append('.')`?